packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -1440,10 +1409,25 @@ describe('src/cy/commands/navigation', () => {
       it('throws when attempting to visit a 2nd domain on different port', function (done) {
```

Comment:
This test and the following 3 are failing for me in both open and run mode locally with the flag turned off, though I see they are succeeding in CI.  Not sure why:

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/4873279/164559358-9ef023e5-71ba-46e9-a87e-2e66f684a916.png">

Reply:
Disregard this, I was on a stale build.